data/lang/module-newseventcommodity/en.json

```diff
@@ -53,15 +53,15 @@
   },
   "FLAVOUR_13_NEWSBODY": {
     "description": "Imply that nobody wants to buy ROBOTS anymore.",
-    "message": "It is now man against machine in the {system} system ({sectorx}, {sectory}, {sectorz}), as \"behavioural malfunction\" causes home consumers to lose faith in robot servants."
+    "message": "It is now man against machine in the {system} system ({sectorx}, {sectory}, {sectorz}), as \"behavioral malfunction\" causes home consumers to lose faith in robot servants."
```

Comment:
Not spelling error, we're a clone of a UK game, and most of the native speakers have been from UK.

Reply:
That's fine, I'll drop that commit